### ISSUE TITLE
Whitelisting no longer works in Tailwind Options

### ISSUE THREAD

**Issue by jhull** [nuxt-community/tailwindcss-module#124]
What worked in prior builds of Nuxt and TailwindCSS in the purgeCSS key in `nuxt.config.js`:
```
      whitelist: ['tip', 'warning', 'expert'],
      whitelistPatternsChildren: [/content.$/]
```
No longer works in the updated built-in options key for Tailwind:

```
  purge: {
    options: {
      whitelist: ['tip', 'warning', 'expert'],
      whitelistPatternsChildren: [/content.$/]
   }
}
```

The only thing that does work is to set `enabled: false` (contrary to this issue)--but then CSS is not purged.

**Comment by appinteractive** [comment]
have the same issue

**Comment by hecktarzuli** [comment]
I have this issue as well, and it seems the purge comments don't even work anymore.

/* purgecss start ignore */

/* purgecss end ignore */

These worked fine versions < 1.14

**Comment by appinteractive** [comment]
@hecktarzuli What do you mean with `placeholder styles`?

**Comment by bpeab** [comment]
I am having the same issues, especially the start/end ignore which don't work at all.

**Comment by hecktarzuli** [comment]
@appinteractive 

tailwind.css
.blah { background-color: red;}

layouts/default.vue
<style>.blah { background-color: yellow;} </style>

And nothing really USES blah in the template

Result:
FINALLY .blah shows up in the page, and it's red!  Kinda weird.

**Comment by appinteractive** [comment]
@hecktarzuli uff thats really strange, I also did copy a whole styles into my components just to get it working but thats even strager

**Comment by jhull** [comment]
Noticed the last issue closed was three weeks ago - and no comments here for over a week. Is this still being actively maintained?

**Comment by wallopthecat** [comment]
It wasn't closed, it was marked as a bug

**Comment by jhull** [comment]
I meant the last time anyone touched this module was 3 weeks ago. It seems like enough people are experiencing issues with purgecss and I'm not sure if it's being addressed.

**Comment by wallopthecat** [comment]
I agree and misunderstood your comment to be referring to this issue, not issues in general

**Comment by leon0399** [comment]
same issue with [vue-awesome-swiper](https://github.com/surmon-china/vue-awesome-swiper) via SSR configuration using directive

**Comment by appinteractive** [comment]
@Atinux any thoughts on that topic?

**Comment by Atinux** [comment]
Can you please create a reproduction on CodeSandBox so I can quickly look at it and patch if necessary?

**Comment by nivv** [comment]
So that's why my app.js is 5 MB all of a sudden. Pulling in fontawesome it seems

**Comment by Atinux** [comment]
Thank you @ianchak 

Also, please take a look at https://tailwindcss.com/docs/controlling-file-size#removing-all-unused-styles

**Comment by kswedberg** [comment]
Not sure if this is the source of the problem, but I just noticed that PurgeCSS changed their `blacklist` and `whitelist` options to `safelist` and `blocklist`.

**Comment by AndreasElia** [comment]
Also having this issue, the above comment is rather interesting as a possible cause though.

**Comment by c3s4r** [comment]
I just confimed t works if you use `options` and `safelist`:

```
purge: {
  options: {
    safelist: ["whitelisted"],
  },
  // ...
}
```

**Comment by Atinux** [comment]
It would be awesome if anyone feel like opening a PR to add a block about whitelisting classes when using plugins on https://tailwindcss.nuxtjs.org/tailwind-config

**Comment by redbastie** [comment]
Just wanted to confirm @c3s4r solution is working for me! Use `safelist` not `whitelist` in your options.
